UncomplicatedCustomItems/Plugin.cs

```diff
@@ -38,6 +35,8 @@ public override void OnDisabled()
             Events.Internal.Player.Unregister();
             Events.Internal.Server.Unregister();
 
+            Instance = null;
```

Comment:
Do not do that, cuz other plugins to be call this Instance in OnDisabled will be throw exception

Reply:
APIs are now not anymore bound with the Plugin class, so the istance is only internal